### DISCUSSION TITLE
Better website management

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2473] [Ideas]
* The website does not yet have support for touchscreen devices.
* You cannot rename your username because this is part of the login entry (this can be resolved with a nickname system).
* You cannot delete your cartridges on the website.
* You can create a account in the tic-80 website, but you don't have 100% control over it.
* It is not possible to delete your account.
* You cannot report comments on the website.
* You cannot report users on the website.
* You cannot report cartridges on the website.

**Comment by The-Beesh-Spweesh** [comment]
You also cannot edit or delete comments you have posted on the website.
